src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1393,6 +1393,13 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode)
                                 ->scalarNode('ciphers')
                                     ->info('A list of SSL/TLS ciphers separated by colons, commas or spaces (e.g. "RC3-SHA:TLS13-AES-128-GCM-SHA256"...)')
                                 ->end()
+                                ->scalarNode('buffer')
+                                    ->info('Response content should be buffered or not')
```

Comment:
Whether the response content should be buffered or not or a regex telling which content-types should be

Reply:
Changed the info content